packages/amplify-e2e-tests/src/__tests__/help.test.ts

```diff
@@ -24,10 +24,26 @@ describe('help happy paths', () => {
   });
 
   test('amplify status', async () => {
-    await statusWithHelp(projRoot, ['USAGE', 'amplify status [-v | --verbose]']);
+    await statusWithHelp(projRoot);
```

Comment:
what is this testing here? We should have an expect statement that we are testing against

Reply:
The chain.wait expects a certain output. If that output is not given, then the tests will fail.